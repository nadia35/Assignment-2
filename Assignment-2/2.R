getMaxMilk <- function(cow1, cow2, cow3){
  total_milk <- c(cow1, cow2, cow3)
  max_milk <- max(total_milk, na.rm = TRUE)
  return(max_milk)
}
