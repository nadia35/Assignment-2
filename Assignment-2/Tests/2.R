source("2.R")

print(getMaxMilk(8, 12, 15))
print(getMaxMilk(8.2, 12.3, 15.4))
print(getMaxMilk(99, 1, 0))
print(getMaxMilk(33, NA, 10))
print(getMaxMilk(NA, 2.9, 17.6))