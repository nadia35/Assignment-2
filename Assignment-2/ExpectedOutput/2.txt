[1] 15
[1] 15.4
[1] 99
[1] 33
[1] 17.6
